crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -492,18 +491,29 @@ pub fn argmax_config(node: &Node) -> (usize, usize, usize) {
     for (key, value) in node.attrs.iter() {
         match key.as_str() {
             "axis" => axis = value.clone().into_i64(),
-            "select_last_index" => select_last_index = value.clone().into_i64(),
+            "select_last_index" => log::warn!(
+                "select_last_index param for argmax is ignored in burn (got {:?})",
+                value
+            ),
```

Comment:
I think we can still capture the `select_last_index` value here, but only warn if it is 1 (because the default implementation pretty much everywhere including Burn is to return the first max value, not last).